docs/userguide/entry_point.rst

```diff
@@ -4,15 +4,26 @@
 Entry Points
 ============
 
-Packages may provide commands to be run at the console (console scripts),
-such as the ``pip`` command. These commands are defined for a package
-as a specific kind of entry point in the ``setup.cfg`` or
-``setup.py``.
+Entry points are a very useful feature of Python packages.
+They come in use in two scenarios:
```

Comment:
I think we can benefit here for a more specific definition of what is an entry-point. What do you think about the following change?

```suggestion
Entry points are a type of metadata that can be exposed by packages on installation.
They are a very useful feature of the Python ecosystem,
and come specially handy in two scenarios:
```

Reply:
Sure, makes sense!